web/cypress/e2e/app.cy.ts

```diff
@@ -1,8 +1,16 @@
-describe("ARC Portal UI", () => {
+describe("ARC Portal UI unauthenticated", () => {
   it("shows the ARC portal phrase on the homepage", () => {
     cy.visit("/");
 
     cy.contains("ARC portal").should("be.visible");
     cy.contains("fibble").should("not.exist");
   });
 });
+
+describe("ARC Portal UI authenticated", () => {
+  it("can be logged into as an admin", () => {
+    cy.loginAsAdmin();
+    cy.visit("/");
+    cy.contains("Login").should("not.exist");
+  });
+});
```

Comment:
This test still passes even if you pass in an incorrect password. In this case, the page still redirects back to the home page but the user has not been logged in successfully.